### DISCUSSION TITLE
Melted edges in imagick interface

### DISCUSSION THREAD

**Discussion by jasonwilliamsau** [chillerlan/php-qrcode#259] [Q&A]
My QR codes are using a logo or background image but would like the `melted` edges like in the example https://github.com/chillerlan/php-qrcode/blob/main/examples/svgMeltedModules.php

Is this achievable?

**Comment by codemasher** [comment]
There is a similar example for GD, but i haven't gotten around to port this for ImageMagick yet: https://github.com/chillerlan/php-qrcode/blob/main/examples/imageWithRoundedShapes.php
